docs/pages/Cardinal/Cross_Shard_Communication/evm_to_cardinal.mdx

```diff
@@ -0,0 +1,188 @@
+import { Callout } from 'nextra/components'
+
+<Callout type={"warning"}>
+    EVM to Cardinal communication is an experimental feature. The API is not final and bugs are expected to be encountered.
+</Callout>
+
+
+# EVM To Cardinal Communication
+
+The EVM base shard provides a precompile, called Router, that enables smart contracts to read data from and send messages to Cardinal game shards.
+
+For messages, router operates asynchronously, which requires a callback mechanism to be used. Messages are only forwarded from Router's `sendMessage` method once the EVM transaction that called it successfully executes. Shortly after, the result will be available for consumption via Router's `messageResult` method.
+
+Queries are synchronous and do not require a callback.
+
+## Precompile Address
+
+`0x356833c4666fFB6bFccbF8D600fa7282290dE073`
```

Comment:
I don't understand this section. Questions that come to mind: What is a precompile address? Is this specific value important? Was it generated? How do I generate my own? Where do I use it? 

All these questions don't need to be answered, but at least a link to some place that provides more information would be helpful.

Reply:
added a callout to the top that should suffice. lmk how you feel abt it